types/stampit/v3/index.d.ts

```diff
@@ -0,0 +1,525 @@
+// Type definitions for stampit 3.0
```

Comment:
The appropriate thing is to move `stampit 2.1` to the `v2` folder, and put this `v3` one in the top-level.